Modules/Role/Http/Services/RoleService.php

```diff
@@ -147,9 +128,48 @@ public function delete($id): array
      * @return array
      */
     public function showAdmin(): array
+    {
+
+        $adminUsers = User::where('role', '!=', Role::ROLE_SUPER)
+            ->where('role', '!=', Role::ROLE_BRAND)
+            ->where('role', '!=', Role::ROLE_RETAILER)
+            ->where('role', '!=', 1)
+            ->with('roles')->get();
```

Comment:
Change this query to:
$excludedRoles = [Role::ROLE_SUPER, Role::ROLE_BRAND, Role::ROLE_RETAILER, 1];
$adminUsers = User::whereNotIn('role', $excludedRoles)
    ->with('roles')
    ->get();
    
    Try to utilize Laravel Eloquent as much as possible.

Reply:
Fixed